make.js

```diff
@@ -111,7 +96,7 @@ target.clean = function () {
 // ex: node make.js gendocs
 // ex: node make.js gendocs --task ShellScript
 //
-target.gendocs = function() {
+CLI.gendocs = function(/** @type {{ suite: string; node: string; … }} */ argv) {
```

Comment:
```suggestion
CLI.gendocs = function() {
```
argv not used in this function